pkg/agent/config_test.go

```diff
@@ -1,15 +1,202 @@
 package agent
 
 import (
+	"bytes"
 	"fmt"
+	"io"
+	"log"
+	"os"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/d4l3k/messagediff"
+	"github.com/jetstack/preflight/pkg/client"
 	"github.com/kylelemons/godebug/diff"
-	"gopkg.in/d4l3k/messagediff.v1"
+	"github.com/stretchr/testify/assert"
 )
 
+func TestGetConfiguration(t *testing.T) {
+	t.Run("minimal successful configuration", func(t *testing.T) {
+		got, cl, err := getConfiguration(discardLogs(t),
+			Config{Server: "http://localhost:8080", Period: 1 * time.Hour},
```

Comment:
**Self-review:** This URL isn't actually used, I guess I should have picked a clearer URL like `https://fake`.

Reply:
Done. I've renamed it to `http://api.venafi.eu`.